Makefile

```diff
@@ -66,10 +66,81 @@ build-all:
 aro: check-release generate
 	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro
 
+# Target to deploy the CosmosDB database and container
+.PHONY: deploy-database
+deploy-database:
+	@echo "Deploying CosmosDB database and container..."
+	@az deployment group create \
+		-g "$(RESOURCEGROUP)" \
+		-n "databases-development-$(USER)" \
+		--template-file pkg/deploy/assets/databases-development.json \
+		--parameters \
+			databaseAccountName="$(DATABASE_ACCOUNT_NAME)" \
+			databaseName="$(DATABASE_NAME)" \
+		1>/dev/null || echo "Deployment failed or already exists"
+
+# Target to login to ACR
+.PHONY: acr-login
+acr-login:
+	@echo "Logging into ACR..."
+	@echo "PULL_SECRET: $${PULL_SECRET}"
+	@acr_auth=$$(echo "$${PULL_SECRET}" | jq -r '.auths["arointsvc.azurecr.io"].auth' | base64 -d); \
+	echo "Decoded ACR Auth: $$acr_auth"; \
+	username=$$(echo "$$acr_auth" | cut -d: -f1); \
+	password=$$(echo "$$acr_auth" | cut -d: -f2); \
+	echo "ACR Username: $$username"; \
+	echo "ACR Password: $$password"; \
+	docker login arointsvc.azurecr.io --username $$username --password $$password
+
+# Target to run the local RP
 .PHONY: runlocal-rp
-runlocal-rp:
-	go run -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro rp
-
+runlocal-rp: ci-rp deploy-database acr-login 
```

Comment:
Could we remove `deploy-database` from the prerequisites here? It takes about 75 seconds simply to refresh a database that doesn't need any changes, which is a long time to wait each time you want to start your local RP instance.

I played around with a [what-if deployment](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-what-if?tabs=azure-powershell) to see if it would allow us to keep the prerequisite while speeding things up, but I concluded that that's more trouble than it's worth.